src/script/chem/molfile/v3000.js

```diff
@@ -162,6 +162,8 @@ function v3000parseSGroup(ctab, ctabLines, sgroups, atomMap, shift) { // eslint-
 			sg.data.connectivity = props['CONNECT'][0].toLowerCase();
 		if (props['FIELDDISP'])
 			sGroup.applyDataSGroupInfo(sg, stripQuotes(props['FIELDDISP'][0]));
+		if (props['FIELDINFO'])
+			sg.data.units = props['FIELDINFO'][0];
```

Comment:
noticed from Marvin that the units for a Data S-Group are written out as FIELDINFO=<units>